roles/pulp_common/tasks/install_pip.yml

```diff
@@ -145,6 +145,8 @@
         virtualenv: '{{ pulp_install_dir }}'
         virtualenv_command: '{{ pulp_python_interpreter }} -m venv'
       when: pulp_source_dir is undefined
+      environment:
+        SETUPTOOLS_USE_DISTUTILS: stdlib
```

Comment:
I see the commit message and I understand why we are doing this, but I really want a comment above this line.